integration/container/exec_test.go

```diff
@@ -139,3 +139,38 @@ func TestExecUser(t *testing.T) {
 
 	assert.Assert(t, is.Contains(result.Stdout(), "uid=1(daemon) gid=1(daemon)"), "exec command not running as uid/gid 1")
 }
+
+func TestContainerExecKillNoSuchExec(t *testing.T) {
```

Comment:
Can we combine this test with the other one and run it as a subtest or just as the beginning?
Otherwise the test names are too close to single out.